media-source/mediasource-appendbuffer-quota-exceeded.html

```diff
@@ -1,10 +1,14 @@
 <!DOCTYPE html>
 <!-- Copyright © 2016 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
 <meta charset="utf-8">
+<meta name="timeout" content="long">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="mediasource-util.js"></script>
 <script>
+    var subType = MediaSourceUtil.getSubType(MediaSourceUtil.VIDEO_ONLY_TYPE);
```

Comment:
hmm. I missed this; shouldn't be a problem, but I'll s/VIDEO/AUDIO/ on this line.